### PR TITLE
T&A #41600: Change byline of Test Setting "Introductory message"

### PR DESCRIPTION
**Pull request by dsstrassner**
for ilias_de & ilias_en.lang. Please cherry pick also to Trunk.

### CODE REVIEW

**Comment by dsstrassner** [comment]
The feature request states as the second part of the Byline: "The message can be edited in the sub-tab "Introductory Message" of these settings and is accessible already before the test can be started." - The sub-tab part is missing, but the JourFixe decision has no information whether this byline part is okay or not. So, I changed only the "info" to "test"-tab. Maybe @matthiaskunkel could state, if we should add the "sub-tab" info also ;-)

File: lang/ilias_de.lang [hunk 1]
```diff
@@ -1515,7 +1515,7 @@ assessment#:#tst_instant_feedback_trigger_forced_desc#:#Durch Beantwortung einer
 assessment#:#tst_instant_feedback_trigger_manual#:#Teilnehmer können die Rückmeldung selbst auslösen
 assessment#:#tst_instant_feedback_trigger_manual_desc#:#Die Rückmeldung steht Teilnehmern zur Verfügung, wird aber erst gezeigt, wenn der Teilnehmer diese auslöst.
 assessment#:#tst_introduction#:#Einleitung
-assessment#:#tst_introduction_desc#:#Zeigt einen Einleitungstext auf dem 'Info'-Reiter des Tests an. Der Text ist bereits vor Beginn des Tests zugänglich.
+assessment#:#tst_introduction_desc#:#Zeigt einen Einleitungstext auf dem 'Test'-Reiter an. Der Text ist bereits vor Beginn des Tests zugänglich.
```
Comment (matthiaskunkel) [inline lang/ilias_de.lang:1518]: Please use German Anführungszeichen: „Test“-Reiter.

**Review by matthiaskunkel — CHANGES_REQUESTED** [review]

**Comment by matthiaskunkel** [comment]
And adding an additional information about where the introductory message is created would be fine for me.

**Comment by dsstrassner** [comment]
Done ;-)

**Comment by thojou** [comment]
Hey @dsstrassner,

i took your PR and integrated the changes into trunk.

Thank you.

**Pull request merged**